app/src/androidTest/java/org/mozilla/tv/firefox/ui/BasicNavigationTest.kt

```diff
@@ -134,4 +135,14 @@ class BasicNavigationTest {
             assertActivityFinishing(activityTestRule)
         }
     }
+    @Test
+    fun screenNavigationFocusTest() {
```

Comment:
I wonder if focus testing might deserve its own class entirely.

Reply:
I took Navigation to mean "navigation within the app" so in this case, I put it here to handle navigation to other fragments.